source/PokerSetup_2.py

```diff
@@ -0,0 +1,341 @@
+        self.name = name
+        self.hand = []
+        self.knownCards = []
+        self.value = []
+        self.win = []
+        self.money = money
+
+    def highCard(self, value):
+        #   Player's Best hand is a high card.
+        cardList = []
+        for x in self.hand:
+            cardList.append(x.value)
+        Sort = sorted(cardList) 
+        print("High card is:", Sort[len(Sort) - 1])
+
+    def getPairs(self, value):
+        #   Player's Best hand is a pair.
+        cardList = []
+        duplicate = []
+        x = 0
+        for y in self.hand:
+            cardList.append(y.value)
+            x += 1
+        Sort = sorted(cardList)
+
+        a, b = 0, 1
+        for z in range(x - 1):
+            if Sort[a] == Sort[b]:
+                duplicate.append(Sort[a])
+            a, b = a + 1, b + 1
+
+        if len(duplicate) != 0:
+            print("Highest pair is:", duplicate[len(duplicate) - 1])
+
+    def get2Pairs(self, value):
+        #   Player's Best hand is 2 pairs.
+        cardList = []
+        duplicateList = []
+        pairList = []
+        v = 0
+        for w in self.hand:
+            cardList.append(w.value)
+            v += 1
+        Sort = sorted(cardList)
+
+        x = 0
+        a, b = 0, 1
+        for y in range(v - 1):
+            if Sort[a] == Sort[b]:
+                duplicateList.append(Sort[a])
+                x += 1
+            a, b = a + 1, b + 1
+        Duplicate = sorted(duplicateList)
+
+        if x >= 2:
+            c, d = 0, 1
+            for z in range(x - 1):
+                if Duplicate[c] != Duplicate[d]:
+                    pairList.append(Duplicate[c])
+                c, d = c + 1, d + 1
+            pairList.append(Duplicate[c])
+            
+        if len(pairList) == 2:
+            print("Highest 2 pairs is:", pairList[len(pairList) - 1],
+                  "and", pairList[len(pairList) - 2])
+
+    def threeOfAKind(self, value):
+        #   Player's Best hand is a 3 of a kind.
+        cardList = []
+        duplicateList = []
+        pairList = []
+        self.winNumber = []
+        v = 0
+        for w in self.hand:
+            cardList.append(w.value)
+            v += 1
+        Sort = sorted(cardList)
+
+        x = 0
+        a, b = 0, 1
+        for y in range(v - 1):
+            if Sort[a] == Sort[b]:
+                duplicateList.append(Sort[a])
+                x += 1
+            a, b = a + 1, b + 1
+        Duplicate = sorted(duplicateList)
+
+        if x >= 2:
+            c, d = 0, 1
+            for z in range(x - 1):
+                if Duplicate[c] == Duplicate[d]:
+                    pairList.append(Duplicate[c])
+                c, d = c + 1, d + 1
+        threeOfAKind = sorted(pairList)
+
+        length = len(threeOfAKind) - 1
+        if length >= 0:
+            print("Highest 3 of a kind is:", threeOfAKind[length])
+
+    def evaluatePlayerHand(self):
+        for players in PlayersPlayingRoundList:
```

Comment:
PlayersPlayingRoundList is a List of player objects, so a for loop would go through each individual player. The name of this variable is a little misleading.